pagopa/pn-frontend
# Feat/pn 2138

## Short description
Show skeleton on first page loading

## List of changes proposed in this pull request
It has been created the new component LoadingPageWrapper. This component wraps those pages that should not be shown until they are fully loaded. When tha page is loaded, through the usage of the useState hook, it tells to the component that it is ready and the content is shown

## How to test
Go to Notification, NotificationDetail, Contacts and Deleghe pages and check that the content is shown only when all data are loaded
